File: KantaPe_Web/Kruma.KantaPe/Kruma.KantaPe.Entidad/AperturaUsuario.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Runtime.Serialization;
using System.Text;
using System.Threading.Tasks;

namespace Kruma.KantaPe.Entidad
{
    /// <summary>AperturaUsuario</summary>
    /// <remarks><list type="bullet">
    /// <item><CreadoPor>Creado por John Castillo</CreadoPor></item>
    /// <item><FecCrea>25-07-2016</FecCrea></item></list></remarks>
    [DataContract]
    [Serializable]
    public class AperturaUsuario
    {
        [DataMember]
        public int? IdLocal { get; set; }
        [DataMember]
        public int? IdApertura { get; set; }
        [DataMember]
        public int? IdAperturaUsuario { get; set; }
        [DataMember]
        public string IdUsuario { get; set; }
        [DataMember]
        public string Administrador { get; set; }
        [DataMember]
        public decimal? Latitud { get; set; }
        [DataMember]
        public decimal? Longitud { get; set; }
        [DataMember]
        public DateTime? FechaIngreso { get; set; }
        [DataMember]
        public DateTime? FechaFinal { get; set; }
        [DataMember]
        public string Estado { get; set; }
        [DataMember]
        public string NombreCompleto { get; set; }
        [DataMember]
        public Local Local { get; set; }
        [DataMember]
        public Empresa Empresa { get; set; }
        [DataMember]
        public int? Cantidad { get; set; }
        [DataMember]
        public Kruma.Core.Business.Entity.Persona Persona { get; set; }
        [DataMember]
        public string UsuarioCreacion { get; set; }
        [DataMember]
        public DateTime? FechaCreacion { get; set; }
        [DataMember]
        public string UsuarioModificacion { get; set; }
        [DataMember]
        public DateTime? FechaModificacion { get; set; }
        [DataMember]
        public Kruma.Core.Security.Entity.Usuario  Usuario { get; set; }
    }
}

File: app/src/main/java/com/kruma/kanta/entidad/AperturaUsuario.java
package com.kruma.kanta.entidad;

import com.google.gson.annotations.SerializedName;
import com.kruma.core.business.entity.Persona;

import java.util.Date;

/**
 * Created by John on 27/07/2016.
 */

public class AperturaUsuario {

    //region ATRIBUTOS
    @SerializedName("IdLocal")
    private Integer idLocal;
    @SerializedName("IdApertura")
    private Integer idApertura;
    @SerializedName("IdAperturaUsuario")
    private Integer idAperturaUsuario;
    @SerializedName("IdUsuario")
    private String idUsuario;
    @SerializedName("Administrador")
    private String administrador;
    @SerializedName("Latitud")
    private Double latitud;
    @SerializedName("Longitud")
    private Double longitud;
    @SerializedName("FechaIngreso")
    private Date fechaIngreso;
    @SerializedName("FechaFinal")
    private Date fechaFinal;
    @SerializedName("Estado")
    private String estado;
    @SerializedName("NombreCompleto")
    private String nombreCompleto;
    @SerializedName("Local")
    private Local local;
    @SerializedName("Empresa")
    private Empresa empresa;
    @SerializedName("Cantidad")
    private Integer cantidad;
    @SerializedName("UsuarioCreacion")
    private String usuarioCreacion;
    @SerializedName("FechaCreacion")
    private Date fechaCreacion;
    @SerializedName("UsuarioModificacion")
    private String usuarioModificacion;
    @SerializedName("FechaModificacion")
    private Date fechaModificacion;

    @SerializedName("Usuario")
    private Usuario usuario;

    @SerializedName("Persona")
    private Persona persona;
    //endregion

    //region PROPIEDADES


    public Integer getIdLocal() {
        return idLocal;
    }

    public void setIdLocal(Integer idLocal) {
        this.idLocal = idLocal;
    }

    public Integer getIdApertura() {
        return idApertura;
    }

    public void setIdApertura(Integer idApertura) {
        this.idApertura = idApertura;
    }

    public Integer getIdAperturaUsuario() {
        return idAperturaUsuario;
    }

    public void setIdAperturaUsuario(Integer idAperturaUsuario) {
        this.idAperturaUsuario = idAperturaUsuario;
    }

    public String getIdUsuario() {
        return idUsuario;
    }

    public void setIdUsuario(String idUsuario) {
        this.idUsuario = idUsuario;
    }

    public String getAdministrador() {
        return administrador;
    }

    public void setAdministrador(String administrador) {
        this.administrador = administrador;
    }

    public Double getLatitud() {
        return latitud;
    }

    public void setLatitud(Double latitud) {
        this.latitud = latitud;
    }

    public Double getLongitud() {
        return longitud;
    }

    public void setLongitud(Double longitud) {
        this.longitud = longitud;
    }

    public Date getFechaIngreso() {
        return fechaIngreso;
    }

    public void setFechaIngreso(Date fechaIngreso) {
        this.fechaIngreso = fechaIngreso;
    }

    public Date getFechaFinal() {
        return fechaFinal;
    }

    public void setFechaFinal(Date fechaFinal) {
        this.fechaFinal = fechaFinal;
    }

    public String getEstado() {
        return estado;
    }

    public void setEstado(String estado) {
        this.estado = estado;
    }

    public String getNombreCompleto() {
        return nombreCompleto;
    }

    public void setNombreCompleto(String nombreCompleto) {
        this.nombreCompleto = nombreCompleto;
    }

    public Local getLocal() {
        return local;
    }

    public void setLocal(Local local) {
        this.local = local;
    }

    public Empresa getEmpresa() {
        return empresa;
    }

    public void setEmpresa(Empresa empresa) {
        this.empresa = empresa;
    }

    public Integer getCantidad() {
        return cantidad;
    }

    public void setCantidad(Integer cantidad) {
        this.cantidad = cantidad;
    }

    public String getUsuarioCreacion() {
        return usuarioCreacion;
    }

    public void setUsuarioCreacion(String usuarioCreacion) {
        this.usuarioCreacion = usuarioCreacion;
    }

    public Date getFechaCreacion() {
        return fechaCreacion;
    }

    public void setFechaCreacion(Date fechaCreacion) {
        this.fechaCreacion = fechaCreacion;
    }

    public String getUsuarioModificacion() {
        return usuarioModificacion;
    }

    public void setUsuarioModificacion(String usuarioModificacion) {
        this.usuarioModificacion = usuarioModificacion;
    }

    public Date getFechaModificacion() {
        return fechaModificacion;
    }

    public void setFechaModificacion(Date fechaModificacion) {
        this.fechaModificacion = fechaModificacion;
    }

    public Usuario getUsuario() {
        return usuario;
    }

    public void setUsuario(Usuario usuario) {
        this.usuario = usuario;
    }


    public Persona getPersona() {
        return persona;
    }

    public void setPersona(Persona persona) {
        this.persona = persona;
    }
}
